src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/49.diff.sql

```diff
@@ -0,0 +1,587 @@
+   ,Code1                     varchar(256) COLLATE Latin1_General_100_CS_AS NOT NULL
+   ,CodeOverflow1             varchar(max) COLLATE Latin1_General_100_CS_AS NULL
+   ,SystemId2                 int      NULL
+   ,Code2                     varchar(256) COLLATE Latin1_General_100_CS_AS NOT NULL
+   ,CodeOverflow2             varchar(max) COLLATE Latin1_General_100_CS_AS NULL
+)
+GO
+IF NOT EXISTS (SELECT * FROM sys.types WHERE name = 'TokenTextList')
+CREATE TYPE dbo.TokenTextList AS TABLE
+(
+    ResourceTypeId           smallint NOT NULL
+   ,ResourceSurrogateId      bigint   NOT NULL
+   ,SearchParamId            smallint NOT NULL
+   ,Text                     nvarchar(400) COLLATE Latin1_General_CI_AI NOT NULL
+
+   --PRIMARY KEY (ResourceTypeId, ResourceSurrogateId, SearchParamId, Text)
+)
+GO
+IF NOT EXISTS (SELECT * FROM sys.types WHERE name = 'TokenStringCompositeSearchParamList')
+CREATE TYPE dbo.TokenStringCompositeSearchParamList AS TABLE
+(
+    ResourceTypeId            smallint NOT NULL
+   ,ResourceSurrogateId       bigint   NOT NULL
+   ,SearchParamId             smallint NOT NULL
+   ,SystemId1                 int      NULL
+   ,Code1                     varchar(256) COLLATE Latin1_General_100_CS_AS NOT NULL
+   ,CodeOverflow1             varchar(max) COLLATE Latin1_General_100_CS_AS NULL
+   ,Text2                     nvarchar(256) COLLATE Latin1_General_100_CI_AI_SC NOT NULL
+   ,TextOverflow2             nvarchar(max) COLLATE Latin1_General_100_CI_AI_SC NULL
+)
+GO
+IF NOT EXISTS (SELECT * FROM sys.types WHERE name = 'TokenSearchParamList')
+CREATE TYPE dbo.TokenSearchParamList AS TABLE
+(
+    ResourceTypeId           smallint NOT NULL
+   ,ResourceSurrogateId      bigint   NOT NULL
+   ,SearchParamId            smallint NOT NULL
+   ,SystemId                 int      NULL
+   ,Code                     varchar(256) COLLATE Latin1_General_100_CS_AS NOT NULL
+   ,CodeOverflow             varchar(max) COLLATE Latin1_General_100_CS_AS NULL
+)
+GO
+IF NOT EXISTS (SELECT * FROM sys.types WHERE name = 'TokenQuantityCompositeSearchParamList')
+CREATE TYPE dbo.TokenQuantityCompositeSearchParamList AS TABLE
+(
+    ResourceTypeId           smallint NOT NULL
+   ,ResourceSurrogateId      bigint   NOT NULL
+   ,SearchParamId            smallint NOT NULL
+   ,SystemId1                int      NULL
+   ,Code1                    varchar(256) COLLATE Latin1_General_100_CS_AS NOT NULL
+   ,CodeOverflow1            varchar(max) COLLATE Latin1_General_100_CS_AS NULL
+   ,SystemId2                int      NULL
+   ,QuantityCodeId2          int      NULL
+   ,SingleValue2             decimal(18,6) NULL
+   ,LowValue2                decimal(18,6) NULL
+   ,HighValue2               decimal(18,6) NULL
+)
+GO
+IF NOT EXISTS (SELECT * FROM sys.types WHERE name = 'TokenNumberNumberCompositeSearchParamList')
+CREATE TYPE dbo.TokenNumberNumberCompositeSearchParamList AS TABLE
+(
+    ResourceTypeId           smallint NOT NULL
```

Comment:
Fix tabulation.

Reply:
I would prefer not to spend any time on on-off scripts which should go away with adding SQL project.